caiman_napari/algorithms/mcorr.py

```diff
@@ -69,18 +70,44 @@ def load_output(viewer, batch_item: pd.Series):
 
 
 def load_projection(viewer, batch_item: pd.Series, proj_type):
+    print("loading projection")
     path = batch_item['outputs'].item()["mcorr_output"][0]
 
     Yr, dims, T = cm.load_memmap(path)
     MotionCorrectedMovie = np.reshape(Yr.T, [T] + list(dims), order='F')
 
     # Dict for Projections
-    choices = {'mean': np.mean(MotionCorrectedMovie, axis=0),
-               'standard deviation': np.std(MotionCorrectedMovie, axis=0),
-               'max': np.max(MotionCorrectedMovie, axis=0)}
-    MC_Projection = choices[proj_type]
+    choices = {'mean': 'mean',
+               'standard deviation': 'std',
```

Comment:
I'd just rename `standard deviation` to `std` in the GUI

Reply:
Did it :)